<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogFunctionPlotter</class>
 <widget class="QDialog" name="DialogFunctionPlotter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="chartLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Function expression</string>
       </property>
       <property name="buddy">
        <cstring>inputFunction</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputFunction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>x^2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="7">
      <widget class="QLineEdit" name="inputNumPoints">
       <property name="toolTip">
        <string>Number of points used in the evaluation</string>
       </property>
       <property name="text">
        <string>500</string>
       </property>
       <property name="placeholderText">
        <string>500</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Mi&amp;n. Value</string>
       </property>
       <property name="buddy">
        <cstring>inputMinValue</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>No. of &amp;points</string>
       </property>
       <property name="buddy">
        <cstring>inputNumPoints</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ma&amp;x. Value</string>
       </property>
       <property name="buddy">
        <cstring>inputMaxValue</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="inputMinValue">
       <property name="toolTip">
        <string>Function is evaluted between Min. Value and Max. Value</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="inputMaxValue">
       <property name="toolTip">
        <string>Function is evaluted between Min. Value and Max. Value</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="placeholderText">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonUpdate">
     <property name="text">
      <string>&amp;Update Plot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogFunctionPlotter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogFunctionPlotter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
